compysition/actors/xslt.py

```diff
@@ -77,4 +84,13 @@ def _process_error(self, message, event):
         raise MalformedEventData("Malformed Request: Invalid XML")
 
     def transform(self, etree_element):
-        return self.template(etree_element).getroot()
\ No newline at end of file
+        return self.template(etree_element).getroot()
+
+class XSLT(_XSLT):
+    output = XMLEvent
+
+class XMLXSLT(XSLT):
```

Comment:
This seems kind of redundant?

Reply:
I've been thinking about this more and I agree this is probably unnecessary.  Though, in a similar manor, so are all the actors implemented to support xml vs json (s.a. XMLEventAttributeLookupModifier vs EventAttributeLookupModifier) as we ensure conversion before every consume.